src/app/components/body-units/quote.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
This was moved from being its own component to being part of body-units, since nothing else used it. Also simplified it a bit, since there was an image feature that wasn't being used.